content/widgets/smart-sort.md

```diff
@@ -0,0 +1,30 @@
+---
+title: SmartSort
+type: widget
+html: |
+  <div class="ais-SmartSort">
+    <div class="ais-SmartSort-text">
+      <div>We removed some search results to show you the most relevant ones</div>
+    </div>
+    <button class="ais-SmartSort-button">
+      <div>See all results</div>
+    </button>
```

Comment:
```suggestion
    <button class="ais-SmartSort-button">
      <span>See all results</span>
    </button>
```

I'd like to get rid of `span`, but due to the templating logic in IS.js, we had to have it.

Reply:
Same case here for RIS, the `ButtonTextComponent` will be passed here so the user can choose any container

Reply:
What's the default output in the current version?